tests/components/broadlink/test_sensors.py

```diff
@@ -27,7 +27,7 @@ async def test_a1_sensor_setup(hass):
     assert mock_api.check_sensors_raw.call_count == 1
     device_entry = device_registry.async_get_device({(DOMAIN, mock_entry.unique_id)})
     entries = async_entries_for_device(entity_registry, device_entry.id)
-    sensors = {entry for entry in entries if entry.domain == SENSOR_DOMAIN}
+    sensors = [entry for entry in entries if entry.domain == SENSOR_DOMAIN]
```

Comment:
Why do we change the broadlink tests?

Reply:
That might indeed not be needed, I'll double check before mergimg.